### DISCUSSION TITLE
What is the function of Request With Backend ?

### DISCUSSION THREAD

**Discussion by yswlww** [anse-app/anse#58] [General]
![image](https://github.com/anse-app/anse/assets/132208946/8efc7f01-8035-4201-8fc5-ccc478a94ea7)
What is the function of "Request With Backend"

**Comment by ddiu8081** [comment]
By default, all requests connect directly from the browser to the backend service (e.g. api.openai.com). 

However, there are some countries with network blocking (e.g. China) where direct connection to OpenAI's service is not possible. 

Turning this option on will let the request go through Anse proxy and then forward it to the backend, to solve the possible direct connection problem.

**Reply by yswlww** [reply]
I see thanks for reply
